diff --git a/src/components/BlogFooter.vue b/src/components/BlogFooter.vue
index 291d9e943497f48f2a077105b711701198537b05..0551d8aec1bb27320571581f0dbe018e7ebf41c7 100644
--- a/src/components/BlogFooter.vue
+++ b/src/components/BlogFooter.vue
@@ -24,16 +24,16 @@ const renderData = computed(() => {
     const bottomMessageData = [bottomMessage || []].flat()
 
     // version
-    const isLast = idx === flatData.length - 1
-    if ((version !== false && isLast) || version) {
-      const versionItem = typeof version === 'object' ? version : {}
+    // const isLast = idx === flatData.length - 1
+    // if ((version !== false && isLast) || version) {
+    //   const versionItem = typeof version === 'object' ? version : {}
 
-      data.push({
-        name: versionItem?.name || `@sugarat/theme@${packageJSON.version}`,
-        link: versionItem?.link || 'https://theme.sugarat.top/',
-        icon: versionItem?.icon || themeSVG
-      })
-    }
+    //   data.push({
+    //     name: versionItem?.name || `@sugarat/theme@${packageJSON.version}`,
+    //     link: versionItem?.link || 'https://theme.sugarat.top/',
+    //     icon: versionItem?.icon || themeSVG
+    //   })
+    // }
     // copyright
     if (typeof copyright === 'string') {
       data.push({
